src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -375,14 +376,16 @@ def build_actions(
     group: Group,
     project: Project,
     text: str,
-    color: str,
     actions: Sequence[MessageAction] | None = None,
     identity: RpcIdentity | None = None,
-) -> tuple[Sequence[MessageAction], str, str]:
+) -> tuple[Sequence[MessageAction], str, bool]:
```

Comment:
I know you're just changing the type here, but just starting a discussion for us to think about in the future:
- Not a huge fan of unnamed tuples or multiple returns, I think it should be limited to 2 max; if we do beyond that, it's time to use a dataclass/struct
- is there a way we can combine values without having to explicitly return data
   - for example, can the array length be used to determine the true/false?
   - dataclass/structs can help with this as well
   ```
    class Foo:
         bar: str
         whatever: List[int]
         
         def is_okay(self) -> bool:
              return len(self.whatever) > 3
   ```